subprojects/core/build.gradle.kts

```diff
@@ -298,6 +298,8 @@ packageCycles {
 
 tasks.test {
     setForkEvery(200)
+    systemProperty("user.language", "en")
+    systemProperty("user.country", "US")
```

Comment:
I don't think we want to fix this by changing the locale for all the tests.

Instead, we should find a way to fix the XmlTransformerTest to not be sensitive to changes in locale.  e.g., instead of looking for a specific message, we just look for the exception type or a part of the message.

Reply:
Thank you for your review.  I agree with your points.

I’ve reverted the change to the global locale settings and added assertions to check both the exception type of the cause and the locale-independent parts of the message.

https://github.com/gradle/gradle/pull/33639/commits/5c126a21bf6e16904af1640d1fed67330be97965